src/manifold/src/boolean3.cpp

```diff
@@ -93,9 +93,9 @@ SparseIndices Filter11(const Manifold::Impl &inP, const Manifold::Impl &inQ,
                        const SparseIndices &p1q2, const SparseIndices &p2q1) {
   ZoneScoped;
   SparseIndices p1q1(3 * p1q2.size() + 3 * p2q1.size());
-  for_each_n(autoPolicy(p1q2.size()), countAt(0_z), p1q2.size(),
+  for_each_n(autoPolicy(p1q2.size(), 100'000), countAt(0_z), p1q2.size(),
              CopyFaceEdges<false>({p1q2, p1q1, inQ.halfedge_, 0_z}));
-  for_each_n(autoPolicy(p2q1.size()), countAt(0_z), p2q1.size(),
+  for_each_n(autoPolicy(p2q1.size(), 100'000), countAt(0_z), p2q1.size(),
```

Comment:
I never knew you could write `100'000`; it's cool, but I think I still prefer `1e5` if you don't mind.

Reply:
Yeah this is fine. I just saw this method in the pstld repo and thought it was cool :P